packages/vite-plugin-svelte/src/utils/compile.ts

```diff
@@ -18,6 +20,31 @@ const _createCompileSvelte = (makeHot: Function) =>
 		const { emitCss = true } = options;
 		const dependencies = [];
 
+		if (options.stats) {
+			if (options.isBuild) {
+				if (!stats) {
+					// build is either completely ssr or csr, create stats collector on first compile
+					// it is then finished in the buildEnd hook.
+					stats = options.stats.startCollection(`${ssr ? 'ssr' : 'dom'} compile`, {
+						logInProgress: () => false
+					});
+				}
+			} else {
+				// dev time ssr, it's a ssr request and there are no stats, assume new page load and start collecting
+				if (ssr && !stats) {
+					stats = options.stats.startCollection('ssr compile');
+				}
+				// stats are being collected but this isn't an ssr request, assume page loaded and stop collecting
```

Comment:
will this cause anything to print if ssr is disabled? (probably we won't want it to in that case)

I wonder if you could use a hook like `writeBundle` to understand when a request was done being processed?

Reply:
this is for dev, so no writeBundle calls.  During dev we only log ssr stats because thats what we are able to batch up reliably. 
In theory it is possible to use a debounce to call finish on the dom compile but this can get very messy too. As the dom requests visible in the browser network log relate to 1 compile call (each import), these stats would be nice to have, but are less essential than the ssr stats that are currently completely hidden

Reply:
it makes me wonder if we should have a feature request to Vite to add a request ID. but I wouldn't let that block this

Reply:
interesting idea, though i wonder how that works, esp when you do clientside nav in kit. But we can move that off this comment thread.